app/models/student_task.rb

```diff
@@ -48,7 +48,7 @@ def course
   end
 
   def hyperlinks
-    @hyperlinks ||= participant.hyperlinks
+    @hyperlinks ||= participant.team.nil? ? [] : participant.team.hyperlinks
```

Comment:
Yes, In the same model app/models/student_task.rb, method name content_submitted_in_current_stage?, it calls hyperlinks.present? in checking if content has been submitted.